tests/test_browser.py

```diff
@@ -4482,8 +4482,7 @@ def test_fetch_sync_xhr_in_wasm(self):
   @requires_threads
   def test_fetch_sync_xhr_in_proxy_to_worker(self):
     shutil.copyfile(test_file('gears.png'), 'gears.png')
-    self.btest('fetch/sync_xhr.cpp',
-               expected='1',
```

Comment:
I'd expect to see the REPORT_RESULT code removed from he test too, no?

Reply:
It is, isn't it?

Reply:
Actually does proxy-to-worker support getting the exit status the way the other modes do? This test is failing now (https://app.circleci.com/pipelines/github/emscripten-core/emscripten/12270/workflows/ededc579-0e4a-4b17-b1ec-1aa7e3f6563a/jobs/411614)

Reply:
Ah maybe not.   I've not looked at `--proxy-to-worker` at all.. since as far as I know it doesn't really have any users.

I doubt its worth fixing if its not an easy fix.